### DISCUSSION TITLE
Is there a reason stripe or paypal is missing as providers ?

### DISCUSSION THREAD

**Discussion by giannik** [Sina-Soltani/Parbad#249] [Q&A] [answered]
Hello, 
I just discovered this project. Looks very interesting.
I was curious why no stripe or paypal provider is not provided out of the box , beeing the most well know gateway providers.
Thanks
Yiannis

**Comment by Sina-Soltani** [comment, marked as answer]
Hi, thanks.
The reason was, I couldn't find any nice explained documents about implementing them. Even in their own websites and documents.

So if anybody has experience with it and can implement it really correctly then it would be great.

**Reply by giannik** [reply]
Thanks, 
I'll give it a shot once I learn how parbad works.

**Reply by Sina-Soltani** [reply]
If you are interested you can read the documents. Almost everything is documented.
You can also check the other gateways inside the source codes to see how they're implemented.
